composer.json

```diff
@@ -32,16 +32,16 @@
 		"php": "^8.1",
 		"ext-json": "*",
 		"friendsofphp/php-cs-fixer": "^3.16",
-		"symfony/console": "^5.4 || ^6.0",
-		"symfony/filesystem": "^5.4 || ^6.0"
+		"symfony/console": "^5.4 || ^6.0 || ^7.0",
+		"symfony/filesystem": "^5.4 || ^6.0 || ^7.0"
```

Comment:
Should we also Directory switch to ^6.4 instead of ^6.0 to be in line with the Core requirements?

Reply:
That makes sense, thank you! Applied in 1ee8cc8.